spec/lib/api/v3/user_preferences/user_preference_representer_parsing_spec.rb

```diff
@@ -43,7 +43,6 @@
       {
         'notifications' => [
           {
-            'channel' => 'in_app',
             'all' => true,
```

Comment:
`all` should be removed from this spec as it is no longer a valid subscription option.

Reply:
Fair, but the parsing bit only tests that it is correctly mapped into the per-project structs. It doesn't really test what the settings in there are.